anastasiiya1/project-pet-shelter-new
# Feature/update sidebar

## Description of Changes

Describe what changes have been made in this PR:
- The issue of selecting ads of a given category has been resolved, displaying a list of categories with the number of ads for each category
- Created a DropDown component and placed it on the Sidebar. Updated the redux: changed the state and operations, added reducers

## Type of Changes

Choose the type of changes in this PR:

## Related Issues

Automatically close relevant issues by using keywords:
`Closes #<issue-number>`, `Fixes #<issue-number>`

## How to Test

Describe steps to test this PR:
1. Select Our Animal page
2. Select a category from the drop-down list
3. See that the page contains only ads from the selected category

## Screenshots / Demo

If possible, add screenshots or a demo to improve the understanding of changes.

## Checklist


## Additional Information

Add any other context or information relevant to this PR.
